tools/clang_tidy/README.md

```diff
@@ -4,7 +4,7 @@ A wrapper library and program that runs `clang_tidy` on the Flutter engine repo.
 
 ```shell
 # Assuming you are in the `flutter` root of the engine repo.
-dart ./tools/clang_tidy/bin/main.dart
+./tools/clang_tidy/bin/main.dart
```

Comment:
Maybe instead of adding a new script, or adding a shebang to main.dart, we could just update these docs to point at https://github.com/flutter/engine/blob/main/ci/clang_tidy.sh?

```suggestion
./ci/clang_tidy.sh
```

Reply:
Yeah that's an option.

I don't love the discoverability of "the way to easily run clang_tidy is to run it from the CI folder", and it also has some weirdness, for example you won't be able to pass `--src-dir` because it's already provided by the CI script.

Maybe nothing is necessary tbh and I'll just revert this PR :)

Reply:
> Yeah that's an option.
> 
> I don't love the discoverability of "the way to easily run clang_tidy is to run it from the CI folder", and it also has some weirdness, for example you won't be able to pass `--src-dir` because it's already provided by the CI script.

Yeah, agreed it's not great. My thinking is that It could be an improvement to update the docs to reflect the current state of the tree, and then think about improving the discoverability separately.